CHANGELOG.md

```diff
@@ -2,7 +2,23 @@
 
 ## unreleased
 
+## 3.2.0 RC0 / 2025-01-29
+
+* [FEATURE] remote/otlp: Add feature flag `otlp-deltatocumulative` to support conversion from delta to cumulative. #15165
+* [CHANGE] relabel: Replace actions can now use UTF-8 characters in `targetLabel` field. Note that `$<chars>` or `${<chars>}` will be expanded. This also apply to `replacement` field for `LabelMap` action. #15851
+* [CHANGE] rulefmt: Rule names can use UTF-8 characters, except `{` and `}` characters (due to common mistake checks). #15851
```

Comment:
`[CHANGE]` items should be first in the list.

Reply:
Thanks! Fixed.